### DISCUSSION TITLE
Calendar View see Future Events

### DISCUSSION THREAD

**Discussion by nordy1145** [totaldebug/atomic-calendar-revive#802] [Support]
Hey All,  I was wondering if there is a way to see future events from the calendar itself.  I'm not using a google calendar to backend it, I'm using my calendar from Nextcloud, but when I look at the calendar view I do not see any events unless I click on the day.  Wondering  if it's possible to do?  Image below of the calendar view with nothing showing for the future.
![image](https://user-images.githubusercontent.com/4099083/193088048-4a4efb06-64c4-4e64-adfc-830b0f1edcf5.png)

Thanks!

**Comment by maddox** [comment]
I was hoping for something like this too. What I'd love is a combination of the event view and calendar view at once. 

I'd love the rundown of the next few days that are always showing, with the ability to tap the day of the month to show events for JUST that day. (a bonus would be if it would time out and go back to showing )

**Comment by marksie1988** [comment]
I don't plan to add any new views, that said, if someone else is able to develop a view and provide a pull request I would be willing to merge it into the card.
